Source/GitReleaseManager.Tests/GitReleaseManager.Tests.csproj

```diff
@@ -1,10 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <LangVersion>8.0</LangVersion>
         <TargetFrameworks>net472;netcoreapp2.2</TargetFrameworks>
-        <Title>GitReleaseManager.Tests</Title>
-        <Description>Test Project for GitReleaseManager</Description>
-        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
Currently, we have a shared SolutionInfo.cs file (https://github.com/GitTools/GitReleaseManager/blob/develop/Source/SolutionInfo.cs) that is on each project, as a result, removing this attribute leads to a build failure, since there are duplicate attributes created.  We would either need to remove the shared SolutionInfo.cs file, to add this attribute back in.